spec/amp-var-substitutions.md

```diff
@@ -276,3 +276,12 @@ For instance:
 <amp-pixel src="https://foo.com/pixel?timestamp=TIMESTAMP"></amp-pixel>
 ```
 
+### TOTAL_ENGAGED_TIME
+
+Get the total time the user has been enagaged with the page since the page was
+loaded.
```

Comment:
Loaded, or became visible?

Might be good to be specific here.

cc @cramforce @avimehta for feedback on this.